src/SlmLocale/Strategy/UriPathStrategy.php

```diff
@@ -55,11 +54,11 @@ class UriPathStrategy extends AbstractStrategy
     protected $redirect_to_canonical;
     protected $sl;
     /**
-     * @var TreeRouteStack
+     * @var SimpleRouteStack
      */
     protected $router;
 
-    public function __construct(TreeRouteStack $router = null)
+    public function __construct(SimpleRouteStack $router = null)
```

Comment:
this possibly will fix https://github.com/basz/SlmLocale/issues/94 can you test @koseduhemak